### DISCUSSION TITLE
Understanding "Hold at 3V until C/50"

### DISCUSSION THREAD

**Discussion by Dapid** [pybamm-team/PyBaMM#2617] [Q&A] [answered]
In the experiment definition, one example says: "Hold at 3V until C/50"

What does "until C/50" mean here? I understand charging and discharging at C-numbers, but here?

And does "hold at 3V" mean the battery will be plugged in, with just enough current to compensate for the self-discharge?

**Comment by brosaplanella** [comment, marked as answer]
Hi! It physically means that the voltage is imposed (in this case to 3 V) and thus the current drawn/inputted to the battery is whatever is necessary to hold that voltage. The current will tend to zero as the battery approaches the state-of-charge for that particular voltage, so the C/50 means when to stop (i.e. when the current is smaller than C/50).

**Reply by Dapid** [reply]
Thanks!

That is missing from the docs. I am willing to write it up, where do you think is a good place? The reference or the tutorials? (I am inclined towards the reference).

**Reply by brosaplanella** [reply]
This is a fairly standard step when running experiments on batteries, so maybe the best place would be a clarification on an existing tutorial.
